oc_cdtapi/Dbsm2API.py

```diff
@@ -220,6 +230,14 @@ def wait_for_image(self, audit_id):
         """
         logging.debug('Reached wait_for_image')
         logging.debug('audit_id: [%s]' % audit_id)
+
+        if not isinstance(self.wait_state_timeout, int):
+            logging.debug('wait_state_timeout is not integer, attempting to convert')
+            self.wait_state_timeout = int(self.wait_state_timeout)
```

Comment:
Here we may catch an unwanted exception if conversion fails:
```
>>> int('10a')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '10a'
```
The same goes for next `if`.